### PR TITLE
Updated metadata for 3.0 version of docs (to be merged up to 3.1 and 3.2)

### PR DESCRIPTION
**Pull request by camfindlay**

### CODE REVIEW

File: docs/en/02_Logging_In.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
-<!--
-pagenumber: 1
-title: Introduction & Logging In
--->
+title: Introduction & logging In
+summary: Logging into a SIlverStripe CMS website to start editing content.
```
Comment (camfindlay) [inline docs/en/02_Logging_In.md]: typo

File: docs/en/02_Logging_In.md [hunk 2]
```diff
@@ -1,7 +1,5 @@
-<!--
-pagenumber: 1
-title: Introduction & Logging In
--->
+title: Introduction & logging In
```
Comment (mandrew) [inline docs/en/02_Logging_In.md]: Just a small thing, I wouldn't call this an Introduction. Just call it Logging In. Also change the Title below : )

File: docs/en/04_Site_Management/02_Managing_Roles_And_Permissions.md [hunk 3]
```diff
@@ -1,7 +1,5 @@
-<!--
 title: Roles & permissions
-pagenumber: 2
--->
+summary: Manging CMS access and permissions using groups and roles.
```
Comment (mandrew) [inline docs/en/04_Site_Management/02_Managing_Roles_And_Permissions.md]: Managing

**Comment by camfindlay** [comment]
@mandrew please review

**Pull request merged**